.github/workflows/manual-snapshot-publisher.yml

```
@@ -0,0 +1,873 @@
+name: 📸 Manual Snapshot Publisher
+
+on:
+  workflow_dispatch:
```

actionlint (1.7.7)

maximum number of inputs for "workflow_dispatch" event is 10 but 14 inputs are provided. see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs

(events)